doc/api/http.md

```diff
@@ -1920,10 +1924,13 @@ changes:
 Node.js maintains several connections per server to make HTTP requests.
 This function allows one to transparently issue requests.
 
-`options` can be an object, a string, or a [`URL`][] object. If `options` is a
+`url` can be an a string  or a [`URL`][] object. If `url` is a
```

Comment:
There should only be one space between 'string' and 'or' here.

Reply:
`be an a` -> `be a`